operating/use-admission-controller-enforcement.adoc

```diff
@@ -0,0 +1,65 @@
+[id="use-admission-controller-enforcement"]
+= Using admission controller enforcement
+include::modules/common-attributes.adoc[]
+:context: use-admission-controller-enforcement
+
+toc::[]
+
+[role="_abstract"]
+{product-title} works with https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/[Kubernetes Admission Controllers] and https://docs.openshift.com/container-platform/4.5/architecture/admission-plug-ins.html[{ocp} Admission plug-ins] to allow you to enforce security policies before Kubernetes or {ocp} creates workloads, for example, deployments, daemon sets or jobs.
+{product-title} admission controller prevents users from creating workloads that violate policies you configure in {product-title}.
```

Comment:
Should the sentence start with a "The"? Is there just one admission controller for RHACS?